cmd/dagger/module.go

```diff
@@ -346,6 +346,51 @@ var moduleInstallCmd = &cobra.Command{
 	},
 }
 
+var moduleUpdateCmd = &cobra.Command{
+	Use:     "update [options] <module>",
+	Aliases: []string{"use"},
+	Short:   "Update a dependency",
+	Long:    "Update a dependency to the latest version (or the version specified). The target module must be local.",
+	Example: `"dagger update github.com/shykes/daggerverse/hello@v0.3.0" or "dagger update hello"`,
+	GroupID: moduleGroup.ID,
+	RunE: func(cmd *cobra.Command, extraArgs []string) (rerr error) {
+		ctx := cmd.Context()
+		return withEngine(ctx, client.Params{}, func(ctx context.Context, engineClient *client.Client) (err error) {
+			dag := engineClient.Dagger()
+			modConf, err := getDefaultModuleConfiguration(ctx, dag, true, false)
+			if err != nil {
+				return fmt.Errorf("failed to get configured module: %w", err)
+			}
+			if modConf.SourceKind != dagger.ModuleSourceKindLocalSource {
+				return fmt.Errorf("module must be local")
+			}
+			if !modConf.FullyInitialized() {
+				return fmt.Errorf("module must be fully initialized")
+			}
+
+			modSrc := modConf.Source
+			if len(extraArgs) > 0 {
+				modSrc = modSrc.WithUpdateDependencies(extraArgs)
+			} else {
+				//move this logic to api
```

Comment:
We should make sure that this is done before merging!

Reply:
ha ha. I noticed it as well while reviewing the changes after last push and made the changes locally. will push the fix soon.